### PR TITLE
docs: refactor lazy loading modules example

### PR DESCRIPTION
**Pull request by danielsogl**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

The `angular/http` is deprecated

## What is the new behavior?
Replaced the deprecated `angular/http` module import by `angular/common/http` module

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by danielsogl** [comment]
cc @brandonroberts

File: aio/content/examples/lazy-loading-ngmodules/src/app/app.module.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { BrowserModule } from '@angular/platform-browser';
```
Comment (brandonroberts) [inline aio/content/examples/lazy-loading-ngmodules/src/app/app.module.ts]: I don't feel too strongly about it, but I feel like `BrowserModule` should be first. Its the first import when you generate a new project.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/examples/lazy-loading-ngmodules/src/app/app.module.ts (hunk 1 above)
Comment (danielsogl) [reply aio/content/examples/lazy-loading-ngmodules/src/app/app.module.ts]: But if you read [Style Guide 03-06](https://angular.io/guide/styleguide#style-03-06): 

> Consider listing destructured imported symbols alphabetically. Why? Alphabetizing makes it easier to read and locate symbols.

**Comment by danielsogl** [comment]
@brandonroberts can you remove the cleanup label?

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**